### DISCUSSION TITLE
Make Search case sensitive

### DISCUSSION THREAD

**Discussion by cbutton74** [filamentphp/filament#9697] [Ideas]
Currently the searchable option on a table is **always** case sensitive. Is it possible to specify if a column with searchable() is case sensitive?

For a use case:

If company A stores all its product names as uppercase. Currently every time a user searches for a product they have to make sure they are typing in upper case otherwise no results will show. If the case sensitivity was able to be specified then they would be able to type the product without any issues whether its lowercase, uppercase or a mix of both.

**Comment by sinnbeck** [comment]
If your search is case sensitive that sounds to be because your database is case sensitive. Are you using PostgreSQL?

**Reply by cbutton74** [reply]
Its an MSSQL server which is case sensitive.

**Comment by cbutton74** [comment]
> If your search is case sensitive that sounds to be because your database is case sensitive. Are you using PostgreSQL?

I think I just realised why this is happening. I'm using a model in an external database, which uses the MSSQL server database, where the database for the site is using mySQL that can handle it. Because of the way Laravel deals with the connection I don't think it takes into account the case sensitivity when running the where clause.

I have just tested this on another Filament site that has the MSSQL data base and it seems to work (I have not tried an external database for one of the models though) so I don't think it has anything to do with Filament, which I think @sinnbeck was getting at.

Apologies for wasting your time, this isn't a feature request but a set up issue on my end.

**Reply by sinnbeck** [reply]
Yeah it just makes a plain sql query. It respects whatever case sensitivity the database uses :)

My time wasted. Just happy to help
